packages/patternfly-3/patternfly-react-extensions/src/components/ClassificationBanner/ClassificationBanner.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { Button } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * ClassificationBanner Component for PatternFly React
+ */
+class ClassificationBanner extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      closed: props.closed || false
+    };
+  }
+
+  onClose = () => {
+    this.setState({ closed: true });
+  };
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.closed !== this.state.closed) {
+      this.setState({ closed: nextProps.closed });
+    }
+  }
+
+  render() {
+    const {
+      children,
+      bottomBanner,
+      hostName,
+      userName,
+      closeButton,
+      userNamePosition,
+      hostNamePosition,
+      bannerColor,
+      title,
+      ...props
+    } = this.props;
+
+    const defaultColors = {
+      'pf-red': 1,
+      'pf-blue': 1,
+      'pf-green': 1,
+      'pf-orange': 1,
+      'pf-yellow': 1
+    };
```

Comment:
better to make a constant outside of the class rather than redeclare on every render.

Reply:
@jeff-phillips-18 this should be fixed